projects/plugins/jetpack/json-endpoints/class.wpcom-json-api-list-users-endpoint.php

```diff
@@ -162,11 +163,37 @@ public function callback( $path = '', $blog_id = 0 ) {
 
 		remove_filter( 'user_search_columns', array( $this, 'api_user_override_search_columns' ) );
 
-		$return = array();
+		$return          = array();
+		$include_viewers = (bool) $args['include_viewers'];
+
+		$page    = ( (int) ( $args['offset'] / $args['number'] ) ) + 1;
+		$viewers = $include_viewers ? get_private_blog_users(
```

Comment:
Since this is a WordPress.com function, we probably want to gate it?

Reply:
Done! Gated the whole `$include_viewers` functionality to be under $is_wpcom above, thank you for the find.